eslint.config.mjs

```diff
@@ -0,0 +1,288 @@
+import { fixupConfigRules, fixupPluginRules } from "@eslint/compat";
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
+import prettier from "eslint-plugin-prettier";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+import { createRequire } from "module"
+
+// Workaround for https://github.com/anza-xyz/eslint-plugin-require-extensions/issues/18
+const require = createRequire(import.meta.url)
+const requireExtensions = require("eslint-plugin-require-extensions")
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+const deprecatedForLoanContract = [
+    ['currency', 'brand, asset or another descriptor'],
+    ['blacklist', 'denylist'],
+    ['whitelist', 'allowlist'],
+    ['RUN', 'IST', '/RUN/'],
+  ];
+
+const allDeprecated = [...deprecatedForLoanContract, ['loan', 'debt']];
+
+const deprecatedTerminology = Object.fromEntries(
+Object.entries({
```

Comment:
This isn't formatted correctly. Please make sure this file is included in the `format` and `lint:format` selections